eng/liveBuilds.targets

```diff
@@ -176,6 +176,10 @@
         $(LibrariesNativeArtifactsPath)*.pdb"
         IsNative="true"
         Exclude="@(ExcludeNativeLibrariesRuntimeFiles)" />
+      <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'Browser' and '$(RuntimeFlavor)' == 'CoreCLR'"
+                             Include="
+        $(LibrariesNativeArtifactsPath)pal_random.lib.js;"
```

Comment:
nit; this wrapping looks odd, is that intentional?

Reply:
It is meant to be consistent with the rest of the file:
```
      <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'Browser' and '$(RuntimeFlavor)' == 'CoreCLR'"
                             Include="
        $(LibrariesNativeArtifactsPath)pal_random.lib.js;"
        IsNative="true" />
      <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'Browser' and '$(RuntimeFlavor)' == 'Mono'"
                             Include="
        $(LibrariesNativeArtifactsPath)dotnet.js;
        $(LibrariesNativeArtifactsPath)dotnet.d.ts;
        $(LibrariesNativeArtifactsPath)package.json;
        $(LibrariesNativeArtifactsPath)dotnet.wasm;
        $(LibrariesNativeArtifactsPath)dotnet.js.symbols;
        $(LibrariesNativeArtifactsPath)dotnet.timezones.blat;
        $(LibrariesNativeArtifactsPath)*.dat;"
        IsNative="true" />
```

Reply:
Good to go then, thanks